website/navigation/utils.ts

```diff
@@ -37,8 +41,14 @@ export const getNavItems = async (locale: AppLocale = defaultLocale): Promise<Na
             while (true) {
               try {
                 const fileToTry = filesToTry[currentTry]
-                const { frontmatter }: { frontmatter?: Frontmatter | ((locale: AppLocale) => Frontmatter) } =
-                  await import(`../pages/${fileToTry}`)
+                let frontmatter: Frontmatter | ((locale: AppLocale) => Frontmatter)
+                if (fileToTry.endsWith('.mdx')) {
+                  const { data } = grayMatter(await readFile(join(process.cwd(), 'pages', fileToTry), 'utf8'))
+                  frontmatter = data
+                } else {
+                  const mod = await import(`../pages/${fileToTry}`)
+                  frontmatter = mod.frontmatter
+                }
```

Comment:
the temporal solution, after merging this PR I'll make refactoring to use Nextra's `_meta.json` so this file will be completely removed